app/src/main/java/com/example/mywatchface/mvp/view/WatchFaceView.kt

```diff
@@ -0,0 +1,222 @@
+            random.nextInt(RGB_POSSIBLE_VALUES)
+        )
+    }
+
+    override fun drawWatchFace(canvas: Canvas, mAmbient: Boolean) {
+        val innerTickRadius = mCenterX - INNER_TICK_RADIUS_OFFSET
+        val outerTickRadius = mCenterX
+        for (tickIndex in HOURS) {
+            val tickRot = (tickIndex.toDouble() * ONE_ROUND / HOURS_IN_ANALOG_CLOCK).toFloat()
+            val innerX = sin(tickRot.toDouble()).toFloat() * innerTickRadius
+            val innerY = (-cos(tickRot.toDouble())).toFloat() * innerTickRadius
+            val outerX = sin(tickRot.toDouble()).toFloat() * outerTickRadius
+            val outerY = (-cos(tickRot.toDouble())).toFloat() * outerTickRadius
+            canvas.drawLine(
+                mCenterX + innerX, mCenterY + innerY,
+                mCenterX + outerX, mCenterY + outerY, mTickAndCirclePaint
+            )
+        }
+        val seconds = mCalendar.get(Calendar.SECOND) + mCalendar.get(Calendar.MILLISECOND) / MILLISECONDS
+        val secondsRotation = seconds * FRACTION_ANGLE
+        val minutesRotation = mCalendar.get(Calendar.MINUTE) * FRACTION_ANGLE
+        val hourHandOffset = mCalendar.get(Calendar.MINUTE) / HOUR_HAND_OFFSET
+        val hoursRotation = mCalendar.get(Calendar.HOUR) * OFFSET + hourHandOffset
+        canvas.save()
+        canvas.rotate(hoursRotation, mCenterX, mCenterY)
+        canvas.drawLine(
+            mCenterX,
+            mCenterY - CENTER_GAP_AND_CIRCLE_RADIUS,
+            mCenterX,
+            mCenterY - sHourHandLength,
+            mHourPaint
+        )
+        canvas.rotate(minutesRotation - hoursRotation, mCenterX, mCenterY)
+        canvas.drawLine(
+            mCenterX,
+            mCenterY - CENTER_GAP_AND_CIRCLE_RADIUS,
+            mCenterX,
+            mCenterY - sMinuteHandLength,
+            mMinutePaint
+        )
+        if (!mAmbient) {
+            canvas.rotate(secondsRotation - minutesRotation, mCenterX, mCenterY)
+            canvas.drawLine(
+                mCenterX,
+                mCenterY - CENTER_GAP_AND_CIRCLE_RADIUS,
+                mCenterX,
+                mCenterY - mSecondHandLength,
+                mSecondPaint
+            )
+
+        }
+        canvas.drawCircle(mCenterX, mCenterY, CENTER_GAP_AND_CIRCLE_RADIUS, mTickAndCirclePaint)
+        canvas.restore()
+    }
+
+    private fun tickAndCirclePaint() = Paint().apply {
+        color = mWatchHandColor
+        strokeWidth = SECOND_TICK_STROKE_WIDTH
+        isAntiAlias = true
+        style = Paint.Style.STROKE
+        setShadowLayer(
+            SHADOW_RADIUS, ZERO_FLOAT, ZERO_FLOAT, mWatchHandShadowColor
+        )
+    }
+
+    private fun secondsPaint() = Paint().apply {
+        color = mWatchHandHighlightColor
+        strokeWidth = SECOND_TICK_STROKE_WIDTH
+        isAntiAlias = true
+        strokeCap = Paint.Cap.ROUND
+        setShadowLayer(
+            SHADOW_RADIUS, ZERO_FLOAT, ZERO_FLOAT, mWatchHandShadowColor
+        )
```

Comment:
can this fit in one line?

Reply:
done

Reply:
Pls, if you put all in different s lines, put the value "true" in the next line

Reply:
if i fit that in one line it is going to be too large, so i didnt change this. Also, i moved the true one line below